tools/search/recreate_index.sh

```diff
@@ -22,7 +29,13 @@ done
 npx tsx tools/search/setup.mts
 npx tsx tools/search/recreate_index.mts
 
-ls -l $SEARCH_DB
+# For testing this image outside of Cloud Run.
```

Comment:
I think this is slightly clearer phrasing:

gcp_key_path is for testing this image outside of Cloud Run.
On Cloud Run, permissions are granted via the service account for the job.

Reply:
Will update.